stubs/app/Listeners/ProcessInboundPhoto.php

```diff
@@ -23,7 +23,7 @@ public function handle(UpdateEvent $event)
 
         // Download the largest image to the storage/app directory.
         $photo = collect($update['message']['photo'])->last();
-        $bot->downloadFile($photo, storage_path('app/photos'));
+        $bot->downloadFile($photo['file_id'], storage_path('app/photos'));
```

Comment:
File ID will be retrieved automatically if it's a response object, this is not needed.